cfme/tests/openstack/cloud/test_flavors.py

```diff
@@ -96,15 +121,16 @@ def test_create_instance_with_zero_disk_flavor(new_instance, soft_assert):
 
 
 @pytest.mark.regression
-def test_flavor_crud(appliance, provider, request):
+def test_flavor_crud(appliance, provider, request, is_public):
```

Comment:
Q: are you sure this isn't typo ? I don't see any var or fixture with such name.
UPD: PRT should dispel my doubts in a bit.